frontend/css/free_home_view.css

```diff
@@ -0,0 +1,73 @@
+.color-div {
+    background-color: white;
+}
+
+.color-center {
+    background-color: blue;
+}
+
+/* Extra small devices (portrait phones, less than 576px)*/
+@media (max-width: 575.98px) {
+    .test-div {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=6343113)